### PR TITLE
added test integration methods profile

### PR DESCRIPTION
**Pull request by mariyazaiceva**

### CODE REVIEW

File: src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+@Sql(scripts = "/sql/insert_profile.sql", executionPhase = BEFORE_TEST_METHOD)
+@Sql(scripts = "/sql/delete_profile.sql", executionPhase = AFTER_TEST_METHOD)
+@TestMariaDbContainer
+public class ProfileRepositoryTest {
+
+    @Autowired
+    private ProfileRepository profileRepository;
+
+    @Test
+    void shouldFindById() {
```
Comment (OlgaEvdokimova) [inline src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java:30]: should find what? provide more info for other people

File: src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        final long id = 1;
+        Optional<Profile> byId = profileRepository.findById(id);
```
Comment (OlgaEvdokimova) [inline src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java:32]: byId bad name

File: src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java (hunk 1 above)
Comment (mariyazaiceva) [reply src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java]: Done

File: src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java (hunk 2 above)
Comment (mariyazaiceva) [reply src/test/java/internship/mbicycle/storify/integration/ProfileRepositoryTest.java]: Done

**Pull request merged**